src/plugins/observables.js

```diff
@@ -1,9 +1,40 @@
 /* @flow */
 import isObservable from 'is-observable'
 import StyleRule from '../rules/StyleRule'
-import type {Rule} from '../types'
+import createRule from '../utils/createRule'
+import type {Observable, Rule, RuleOptions, JssStyle} from '../types'
 
 export default {
+  onCreateRule(name: string, decl: JssStyle, options: RuleOptions): Rule | null {
+    if (isObservable(decl)) {
+      // Cast `decl` to `Observable`, since it passed the type guard
+      const props$ = (decl: Observable<{[string]: string}>);
```

Comment:
linter will fail, because I am using no-semicolon style)

Reply:
why renaming decl to props?

Reply:
I would call it decl$

Reply:
I was taking the name from `rule.prop`.  I can change to `decl` if you're prefer, but then you'd end up with shadowing:

```typescript
onCreateRule(name: string, decl: JssStyle, options: RuleOptions): Rule | null {
  // ...
  decl$.subscribe(
    // This is the emission from `decl$`, but might be confused with the argument
    (decl) => {
```

Reply:
rule.prop() is a getter/setter for a property/value, but that object is a declaration - set of properties and values)